modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1484,7 +1477,7 @@ private CompletableFuture<List<Table>> tablesAsyncInternalBusy() {
                             .toArray(CompletableFuture[]::new);
 
                     return CompletableFutures.allOf(tableImplFutures);
-                }), ioExecutor);
```

Comment:
Why is this change needed?

Reply:
The wrapper around SchemaSyncService switches execution to the correct thread, so this would do a second switch